macros-test/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "macros-test"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Small nit :)

Reply:
You'd want this comment removed or what is the nit? :D

Reply:
Yea I think it's best to leave it out as it's just fluff